bundles/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/MagentaTVBindingConstants.java

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.magentatv.internal;
+
+import java.util.Collections;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link MagentaTVBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author markus7017 - Initial contribution
```

Comment:
please provide your real name as an author here

Reply:
I saw your comments and will start to work shortly on those. Driven by the shelly review process I have several topics on the list to change. I let you know when done so it‘s more efficient for you. Thx

Reply:
I saw your comments and will start to work shortly on those. Driven by the shelly review process I have several topics on the list to change. I let you know when done so it‘s more efficient for you. Thx

Reply:
done (all files)